spec/system/scroll_manager_service_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+describe "Ember route-scroll-manager service", type: :system do
+  before do
+    Fabricate(:admin)
+    50.times { Fabricate(:post) }
```

Comment:
🏌️ (dunno if it does anything magical to make the fabrication any faster, but it is shorter 😄)

```suggestion
    Fabricate.times(50, :post)
```